viewer-prototype/src/browser/trace-server-status.ts

```diff
@@ -110,8 +110,14 @@ export class TraceServerConnectionStatusContribution extends DefaultFrontendAppl
 
         try {
             await this.withTimeout(this.traceServerConfigService.startTraceServer(this.path, this.port));
-        } catch {
-            this.messageService.error('Failed to start trace server.');
+        } catch (err) {
+            if (PortBusy.is(err)) {
+                this.messageService.error(
+                    `Error opening serial port ${this.port}. (Port busy)`);
+            } else {
+                this.messageService.error(
```

Comment:
Sometimes it seems the front-end seems to timeout even after successful start in the backend and I get this toaster message. Mabye front-end ping code can be updated to avoid that.

Reply:
Increasing the timeout in withTimeout() below gives the scheduled ping time to clear the serverPending timer in time. Could you please increase it to 12000?